x-pack/plugins/enterprise_search/common/types/indices.ts

```diff
@@ -38,7 +44,8 @@ export interface ConnectorIndex extends ElasticsearchIndex {
 export interface CrawlerIndex extends ElasticsearchIndex {
   crawler: Crawler;
 }
-export interface ElasticsearchIndexWithIngestion extends ElasticsearchIndex {
+export interface ElasticsearchIndexWithIngestion
```

Comment:
I think this is a sign we should either unify the endpoints, or do something like: 
```
export interface ElasticsearchIndex {
  alias: boolean;
  health?: HealthStatus;
  name: IndexName;
  total: {
    docs: {
      count: number;
      deleted: number;
    };
    store: {
      size_in_bytes: string;
    };
  };
}

export interface ConnectorIndex extends ElasticsearchIndex {
  connector: Connector;
}

export interface CrawlerIndex extends ElasticsearchIndex {
  crawler: Crawler;
}

export interface ElasticsearchIndexWithIngestion
  extends ElasticsearchIndex {
  connector?: Connector;
  crawler?: Crawler;
}

export interface SearchIndex extends ElasticsearchIndex {
privileges: {
    read: boolean;
    manage: boolean;
  };
  status: IndicesStatsIndexMetadataState;
}
```
So you're returning a different type forked from ElasticsearchIndex in your endpoint.

Reply:
Will fix.